searchcore/src/vespa/searchcore/proton/matching/matcher.cpp

```diff
@@ -277,7 +279,7 @@ Matcher::match(const SearchRequest &request, vespalib::ThreadBundle &threadBundl
         SearchReply::Coverage & coverage = reply->coverage;
         coverage.setActive(numActiveLids);
         //TODO this should be calculated with ClusterState calculator.
-        coverage.setTargetActive(numActiveLids);
+        coverage.setTargetActive(bucketdb.getNumActiveDocs());
```

Comment:
Can this TODO be removed after these changes?

Reply:
Yes, it can